frontend/lib/verification/verification_qr_scanner_page.dart

```diff
@@ -39,7 +44,19 @@ class VerificationQrScannerPage extends StatelessWidget {
             child: QrCodeScannerPage(
               onCodeScanned: (code) => _handleQrCode(context, code),
             ),
-          )
+          ),
+          if (config.showDevSettings)
+            TextButton(
+              onPressed: () async {
+                final provider = Provider.of<CardDetailsModel>(context, listen: false);
+                final cardDetails = provider.cardDetails!;
+                final generator = OTPGenerator(cardDetails.totpSecretBase32);
+                final code =
+                    encodeVerificationCardDetails(VerificationCardDetails(cardDetails, generator.generateOTP().code));
+                _handleQrCode(context, code);
+              },
+              child: const Text("Self Verify"),
+            )
```

Comment:
I think I'm not a huge fan of putting this here.
Can't we put this into the dev settings?

Reply:
All of this will be refactored soon (hopefully soon). So I'll leave this for now.